### ISSUE TITLE
This would be nice as a nuget library.

### ISSUE THREAD

**Issue by Shadowblitz16** [vercidium-patreon/meshing#4]
Hello I am interested in voxel games and this video pointed me to you..
https://www.youtube.com/watch?v=CJ94gOzKqsM.
 
This would be a good project to upload to nuget as C# voxel library.

I would also love to see more on this.

**Comment by meedo-blip** [comment]
Yes, i am not experienced with c and c++ header file linking and stuff so i found c# to be more easy to open. btw this is coming from a java dev's opinion lol :)

**Comment by Vercidium** [comment]
Since this is a standalone executable I'm not sure how it would be used/imported as a nuget library. Can you please elaborate?

**Comment by Shadowblitz16** [comment]
oh I misunderstood sorry. I thought it was a example on how to do it.
